test/functional/wallet_signrawtransactionwithwallet.py

```diff
@@ -270,21 +270,21 @@ def test_signing_with_missing_prevtx_info(self):
                     }
                 ])
 
-            assert_raises_rpc_error(-3, "Missing vout", self.nodes[0].signrawtransactionwithwallet, rawtx, [
+            assert_raises_rpc_error(-3, "JSON value of type null is not of expected type number", self.nodes[0].signrawtransactionwithwallet, rawtx, [
                 {
                     "txid": txid,
                     "scriptPubKey": pubkey,
                     "amount": 1,
                 }
             ])
-            assert_raises_rpc_error(-3, "Missing txid", self.nodes[0].signrawtransactionwithwallet, rawtx, [
+            assert_raises_rpc_error(-3, "JSON value of type null is not of expected type string", self.nodes[0].signrawtransactionwithwallet, rawtx, [
                 {
                     "scriptPubKey": pubkey,
                     "vout": 3,
                     "amount": 1,
                 }
             ])
-            assert_raises_rpc_error(-3, "Missing scriptPubKey", self.nodes[0].signrawtransactionwithwallet, rawtx, [
+            assert_raises_rpc_error(-3, "JSON value of type null is not of expected type string", self.nodes[0].signrawtransactionwithwallet, rawtx, [
```

Comment:
I don't think this needs to change. The previous error message seems clearer and more direct